### PR TITLE
systemboot: add Quanta S9S to supported OEM product list

### PR DESCRIPTION
**Pull request by timchuquanta**
Signed-off-by: Tim Chu <Tim.Chu@quantatw.com>

### CODE REVIEW

File: cmds/boot/systemboot/main.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ const VpdBmcBootOrderOverride = "bmc_bootorder_override"
 var bmcBootOverride bool
 
 // Product list for running IPMI OEM commands
-var productList = [4]string{"Tioga Pass", "Mono Lake", "Delta Lake", "CraterLake"}
+var productList = [5]string{"Tioga Pass", "Mono Lake", "Delta Lake", "CraterLake", "S9S"}
```
Comment (paulmenzel) [inline cmds/boot/systemboot/main.go:58]: Probably this needs to be extended in the future  to also contain the vendor?

**Review by paulmenzel — COMMENTED** [review]
I am curious. Do you have a URL for the device?

**Comment by timchuquanta** [comment]
> I am curious. Do you have a URL for the device?

You mean the URL to access this device? No.

**Comment by paulmenzel** [comment]
> > I am curious. Do you have a URL for the device?
> 
> You mean the URL to access this device? No.

Yes, that is what I meant. To bad, it’s not public.

**Review by rminnich — APPROVED** [review]

**Pull request merged**